cockpit/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:11-alpine as builder
+RUN apk --no-cache upgrade
+
+WORKDIR /app
+COPY package*.json ./
+RUN npm install
+COPY . .
+RUN npm run compile
+
+FROM node:11-alpine

```

Comment:
You copied it from a non-merged PR, right?

There is no port exposure defined.
The "RUN apk --no-cache upgrade" should happen for the production image, not the build image